spantest/emulator.go

```diff
@@ -144,17 +160,35 @@ func (fx *EmulatorFixture) NewDatabaseFromDDLFiles(t testing.TB, globs ...string
 		}
 	}
 	assert.Assert(t, len(statements) > 0)
-	return fx.NewDatabaseFromStatements(t, statements)
+	return statements
 }
 
 // NewDatabaseFromStatements creates a new database with a random ID from the provided statements.
 func (fx *EmulatorFixture) NewDatabaseFromStatements(t testing.TB, statements []string) *spanner.Client {
+	return fx.newDatabaseFromStatements(t, statements, []byte{})
+}
+
+// NewDatabaseFromStatements creates a new database with a random ID from the provided statements and proto descriptors.
+func (fx *EmulatorFixture) NewDatabaseFromStatementsAndProtoDesc(
```

Comment:
I am wondering how often this is going to change so we end up with having to create more and more methods in the future. Would it be a better idea to rely on functional options in this to avoid having to create yet another method when we want to introduce more changes (for example, if we ever decide to support PostgreSQL)?

Reply:
Yes, I had a similar thought, but this felt much easier to implement. However, I can look into doing it with functional options instead.

Reply:
Pushed a new commit now that changes to functional options. I like it!

Reply:
We can squash the commits before merging.